extern/sector-storage/stores/local.go

```diff
@@ -127,6 +132,25 @@ func (p *path) stat(ls LocalStorage) (fsutil.FsStat, error) {
 		stat.Available = 0
 	}
 
+	if p.maxStorage > 0 {
+		used, err := ls.DiskUsage(p.local)
+		if err != nil {
+			return fsutil.FsStat{}, err
+		}
+
+		stat.Max = int64(p.maxStorage)
+		stat.Used = used
+
+		avail := int64(p.maxStorage) - used
+		if uint64(used) > p.maxStorage {
+			avail = 0
+		}
+
+		if avail < stat.Available {
+			stat.Available = avail
+		}
+	}
+
 	return stat, err
```

Comment:
So it's possible to have `stat.Used > stat.Max`, but that's fine, right? You just can't safely assume that `Available + Used = Max`?

Reply:
Correct